pkg/network/ovn_kubernetes.go

```diff
@@ -91,7 +91,7 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 	data.Data["OVN_service_cidr"] = svcpools
 
 	if c.HybridOverlayConfig != nil {
-		if c.HybridOverlayConfig.HybridClusterNetwork != nil {
+		if c.HybridOverlayConfig.HybridClusterNetwork != nil && len(c.HybridOverlayConfig.HybridClusterNetwork) > 0 {
```

Comment:
ok but if you feel like updating it you can remove the `nil` check, since `len(nil) == 0` so it's redundant

Reply:
good catch. I just updated it.